scripts/coverage_diff.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
+
+# Example usage: python scripts/coverage_diff.py child_build_dir parent_build_dir
+import xml.etree.ElementTree as ET
+import sys
+import argparse
+import glob
+
+parser = argparse.ArgumentParser(description='Optional app description')
+parser.add_argument('child_coverage', type=str,
+                    help='Child branch build directory')
+parser.add_argument('parent_coverage', type=str,
+                    help='Parent branch build directory')
+args = parser.parse_args()
+
+child_cov_file = glob.glob(args.child_coverage + "/Testing/**/Coverage.xml", recursive = True)[0]
+parent_cov_file = glob.glob(args.parent_coverage + "/Testing/**/Coverage.xml", recursive = True)[0]
+
+def want_file(fn):
+    return fn.startswith("./src")
+
+def pprint_table(matrix):
+    s = [[str(e) for e in row] for row in matrix]
+    lens = [max(map(len, col))+1 for col in zip(*s)]
+    fmt = '\t'.join('{{:{}}}'.format(x) for x in lens)
+    table = [fmt.format(*row) for row in s]
+    print('\n'.join(table))
```

Comment:
Nitpick, but you could return a string here and leave the output method (here `print` to stdout) to the caller:
```python
print(pprint_table(matrix))
```